### ISSUE TITLE
Missing lwt.unix dependency

### ISSUE THREAD

**Issue by samoht** [djs55/ocaml-namedpipe#8]
When linking with `named-pipe.lwt` I get:

```
# File "_none_", line 1:
# Error: No implementations provided for the following modules:
#          Lwt_unix referenced from /home/opam/.opam/system/lib/named-pipe/named-pipe_lwt.cmxa(Named_pipe_lwt)
# Command exited with code 2.
```

There seems to indeed be a missing `lwt.unix` in the META file.

**Comment by samoht** [comment]
Sorry wrong repo, I read the build logs wrongly ...

**Comment by djs55** [comment]
Fixed by #10
